regex-assembly/942120.ra

```diff
@@ -3,15 +3,16 @@
 
 ##!+ i
 
+\=\=
```

Comment:
Please add a comment that describes the dialect and function of the operator. `==` is not a valid operator in Postgres, for example.